src/v/cluster/controller_backend.cc

```diff
@@ -380,7 +380,12 @@ controller_backend::bootstrap_ntp(const model::ntp& ntp, deltas_t& deltas) {
     }
 
     auto& first_delta = bootstrap_deltas.front();
-    vlog(clusterlog.trace, "first bootstrap delta of {}: {}", ntp, first_delta);
+    vlog(
```

Comment:
should the commit message say `c/backend: ...` ?

Reply:
Updated.